src/TLuaInterpreter.cpp

```diff
@@ -3203,94 +3203,178 @@ int TLuaInterpreter::hideUserWindow(lua_State* L)
     return 0;
 }
 
+// No documentation available in wiki - internal function
+void TLuaInterpreter::setBorderSize(lua_State* L, int size, int position, bool updateEvent)
+{
+    Host& host = getHostFromLua(L);
+    // position: 0 = top, 1 = right, 2 = bottom, 3 = left
+    switch(position) {
+        case 0: host.mBorderTopHeight = size; break;
+        case 1: host.mBorderRightWidth = size; break;
+        case 2: host.mBorderBottomHeight = size; break;
+        case 3: host.mBorderLeftWidth = size; break;
+    }
+    if (updateEvent) {
+        int x, y;
+        x = host.mpConsole->width();
+        y = host.mpConsole->height();
+        QSize s = QSize(x, y);
+        QResizeEvent event(s, s);
+        QApplication::sendEvent(host.mpConsole, &event);
+    }
+}
+
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setBorderSizes
+int TLuaInterpreter::setBorderSizes(lua_State* L)
+{
+    int sizeTop;
+    if (!lua_isnumber(L, 1)) {
+        lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new top size as number expected, got %s!)", luaL_typename(L, 1));
+        lua_error(L);
+        return 1;
+    } else {
```

Comment:
We can remove the redundant code block here and avoid a *Clazy* **else after return** anti-pattern warning - instead of:
```
    int sizeTop;
    if (!lua_isnumber(L, 1)) {
        lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new top size as number expected, got %s!)", luaL_typename(L, 1));
        return lua_error(L);
    } else {
        sizeTop = lua_tonumber(L, 1);
    }
```
we can use (and also avoid having to have separate declare and assignment operations for `sizeTop`) instead:
```
    if (!lua_isnumber(L, 1)) {
        lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new top size as number expected, got %s!)", luaL_typename(L, 1));
        return lua_error(L);
    }
    int sizeTo = lua_tonumber(L, 1);
```

Reply:
Also found lots occurrences of patterns like this. Not as easy to search/replace though.
Updated my proposed code at least.

Reply:
Yeah another on-going WIP, but I like the way that it avoids declaring the variable until you need it and actually have a value to assign to it. :innocent: